spark/src/test/java/org/apache/iceberg/TestScanTaskSerialization.java

```diff
@@ -0,0 +1,191 @@
+package org.apache.iceberg;
```

Comment:
This file needs the Apache license header, which you can copy from another file.